src/platform/silabs/ConnectivityManagerImpl.h

```diff
@@ -105,7 +105,13 @@ class ConnectivityManagerImpl final : public ConnectivityManager,
     bool _CanStartWiFiScan();
     void _OnWiFiScanDone();
     void _OnWiFiStationProvisionChange();
-#endif
+#if CHIP_DEVICE_CONFIG_ENABLE_SED
+    ConnectivityManager::SEDIntervalsConfig mIntervalsConfig;
```

Comment:
That would cause conflict with the OpenThread sleepy no ?

Reply:
It is enclosed within `CHIP_DEVICE_CONFIG_ENABLE_WIFI_STATION` so it should not affect thread

Reply:
it is building fine on thread with enable_sleep_device flag